Doc/whatsnew/3.11.rst

```diff
@@ -853,6 +853,9 @@ Deprecated
 
   (Contributed by Brett Cannon in :issue:`47061`.)
 
+* Deprecated the *isdst* parameter in :func:`email.utils.localtime`.
+  (Contributed by Alan Williams in :issue:`72346`.)
```

Comment:
```:issue:`72346` ``` will direct to `https://bugs.python.org/issue72346`, which doesn't exist :).  This should now be ```:gh:`72346` ```, IIUC.